server/src/store.js

```diff
@@ -67,6 +68,31 @@ export function putBit(bit) {
   });
 }
 
+export function queryUser({
+  id,
+  twitterId,
+} = {}) {
+  if (id) {
+    return User.findOne({ id: id }).exec();
+  }
+  if (twitterId) {
+    return User.findOne({ 'twitterProfile.id': twitterId }).exec();
+  }
+}
+
+export function putTwitterUser(profile) {
+  const params = {
+    id: uuid.v1(),
+    twitterProfile: profile,
+  };
+
+  return new Promise((resolve, reject) => {
+    User.create(params)
+      .then((data) => resolve(params))

```

Comment:
do you need the params around data?

Reply:
No, and in fact I just installed ESLint and configured VSCode to run it on save. So now there are a bunch more formatting changes that I'm too lazy to split into another PR 😃